easybuild/easyconfigs/a/artic-ncov2019/artic-ncov2019-2021.06.21-foss-2020b.eb

```diff
@@ -0,0 +1,143 @@
+        'source_tmpl': 'importlib_metadata-%(version)s.tar.gz',
+        'checksums': ['34513a8a0c4962bc66d35b359558fd8a5e10cd472d37aec5f66858addef32c1e'],
+    }),
+    ('more-itertools', '8.2.0', {
+        'modulename': 'more_itertools',
+        'checksums': ['b1ddb932186d8a6ac451e1d95844b382f55e12686d51ca0c68b6f61f2ab7a507'],
+    }),
+    ('packaging', '14.5', {
+        'checksums': ['363f9193daa14085b8dfeeb2bf64227bcf1dc85c02ae2a5c6018b01f77e46491'],
+    }),
+    ('args', '0.1.0', {
+        'checksums': ['a785b8d837625e9b61c39108532d95b85274acd679693b71ebb5156848fcf814'],
+    }),
+    ('pytest', '5.4.1', {
+        'checksums': ['84dde37075b8805f3d1f392cc47e38a0e59518fb46a431cfdaf7cf1ce805f970'],
+    }),
+    ('tqdm', '4.45.0', {
+        'checksums': ['00339634a22c10a7a22476ee946bbde2dbe48d042ded784e4d88e0236eca5d81'],
+    }),
+    ('fieldbioinformatics', '1.1.0-rc2', {
+        'modulename': False,
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/artic-network/fieldbioinformatics/archive'],
+        'checksums': [('7083f67b1188e8f9b55f8bd4f7a46853e9c74ae7dde324b1fc8e792a85e58073',
+                       '778ed3cf2f1b0457c696acbed8b99951b2ec042a2035f08bddd95da3884b146b')],
+    }),
+    # This is not upstream version, but artic tweaked one, which is exactly required.
+    ('Porechop', '0.3.2pre', {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/artic-network/Porechop/archive'],
+        'checksums': ['85980d6f37d38a44c66182e7b39bad487211ccfd8cb820c866ceed7ef7a15523'],
+    }),
+    ('binlorry', '1.3.1', {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/rambaut/binlorry/archive'],
+        'checksums': ['001b74cad497b7253b821ceaac8c0b829b2787396a896fc2d3940a704a149b48'],
+    }),
+    ('clint', '0.5.1', {
+        'checksums': ['05224c32b1075563d0b16d0015faaf9da43aa214e4a2140e51f08789e7a4c5aa'],
+    }),
+    ('datrie', '0.8.2', {
+        'checksums': ['525b08f638d5cf6115df6ccd818e5a01298cd230b2dac91c8ff2e6499d18765d'],
+    }),
+    ('PyVCF', '0.6.8', {
+        'modulename': 'vcf',
+        'checksums': ['e9d872513d179d229ab61da47a33f42726e9613784d1cb2bac3f8e2642f6f9d9'],
+    }),
+    ('ont-fast5-api', '3.1.1', {
+        'modulename': 'ont_fast5_api',
+        'checksums': ['ce5a955c5e90a393f040fb36fc461382339fc0b9cd63e3969b9763127dc2b0d3'],
+    }),
+]
+
+components = [
+    (name, version, {
+        'easyblock': 'Tarball',
+        'source_urls': ['https://github.com/artic-network/artic-ncov2019/archive/'],
+        'sources': [{
+            'download_filename': '%s.tar.gz' % local_commit,
+            'filename': SOURCE_TAR_GZ,
+        }],
+        'checksums': ['26bc96742e291795d4a7c1154336715d98168d8ce3524ad7adfea0b5562eb34d'],
+    }),
+]
+local_artic_bins = [
+    'artic', 'artic_fasta_header', 'artic_make_depth_mask', 'artic_mask', 'artic_vcf_filter', 'artic_vcf_merge'
```

Comment:
Where do those binaries come from? The tarball component above doesn't have them, does it?

Reply:
There is actually one more which was missing of the `artic` family.